testgrid/config.yaml

```diff
@@ -357,6 +357,11 @@ test_groups:
   days_of_results: 60
   num_columns_recent: 3
   num_failures_to_alert: 1
+- name: ci-kubernetes-e2e-gce-scale-performance-kube-dns
```

Comment:
in the ideal world, I would say it would probably be better to see both of these jobs in the same tab, but I guess it's not possible? @spiffxp ?
Hopefully, these two jobs, would be exactly the same from results POV, so I would like to avoid, looking into two different tabs to see what is happening...

@kubernetes/sig-scalability-pr-reviews @mborsz @krzysied - for your input about that.

Reply:
@kubernetes/test-infra-reviewers @krzyzacy @BenTheElder @cjwagner - thoughts ^^

Reply:
with which job? Can you achieve that by merging the ginkgo regex?

(also please fix indent)

Reply:
Sorry - let me clarify my question.
With this PR we would have two jobs running exactly the same set of tests (on different days).
But they differ on cluster setup (one is using kube-dns, the other core-dns).
From scalability perspective they will hopefully behave the same way, so if possible I would like to track them on a single testgrid tab. But I'm not sure it's possible...

Reply:
hummm, I don't think so...

You can potentially write some utility to merge the junit results? (given all artifacts are in gcs and you have access...) Or we need to figure out how to up two cluster in one job?

https://k8s-testgrid.appspot.com/sig-node-kubelet#node-kubelet will upload junit with different suffix, and you can see each test case has a distro next to them, maybe something you can reference?

Reply:
So if you look at http://gcsweb.k8s.io/gcs/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet/17941/artifacts/

there are files like:
```
junit_coreos-beta_01.xml
junit_cos-stable1_01.xml
junit_ubuntu_01.xml
```

and testgrid will parse the suffix of junit names and generate rows like
```
test-foo [coreos-beta]
test-foo [cos-stable1]
test-foo [ubuntu_01]
```

Which I believe is similar to what you want to get?

This code is specific to node e2e tests, and they run multiple nodes in the same job. So alternatively, you can generate a gcs entry by merging the two job results :-)

> is it possible to point two different jobs to write to the same gcs directory maybe (so that a single testgrid tab would have results from the two jobs)

Upload path is determined by job name + build ID, and we probably don't want to muck with that.

Reply:
One idea - if evaluating env var is possible, we could simply change the current job to have:

```
--env=CLUSTER_DNS_CORE_DNS=$([ $(date +'%a') == "Wed" ] && echo "false")
```

@krzyzacy - can we do that?

Reply:
how can that help?

Reply:
ah.. that can also work, just you need to output some metadata to tell which dns is on the job?

Reply:
Yes, so we don't need to add a whole new job just to tweak one env.

Reply:
Yes - if evaluating env vars in this yaml works, then that sounds like a perfect solution to me.

Reply:
yeah, I think it works via the `--env` flag, not via container envs

Reply:
@krzyzacy I'm a bit skeptic about that as it seems like we're just passing some key-value strings as env here - https://github.com/kubernetes/test-infra/blob/master/scenarios/kubernetes_e2e.py#L111

Can we easily verify?

Reply:
I know lines like https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml#L155 works? (You can probably run it locally?)